intel_owl/settings/auth.py

```diff
@@ -54,3 +54,15 @@
         "client_id": secrets.get_secret("GOOGLE_CLIENT_ID"),
         "client_secret": secrets.get_secret("GOOGLE_CLIENT_SECRET"),
     }
+
+SITE_ID = 1
+
+# Provider specific settings
+SOCIALACCOUNT_PROVIDERS = {
+    "github": {
+        "APP": {
+            "client_id": "4a7aed5c9ee5a88afb5b",
+            "secret": "a486ba124a601920b559c472631d90a84863f83c",
```

Comment:
If these are the credentials that you are using right now in your test app, I suggest to remove them and rotate them.
These secrets should always be extracted from environment variables, and never hardcoded here, even for testing purposes only. Otherwise it is too easy to risk to push them.

Plus, we would need to add a part in the documentation where we explain how the IntelOwl admin would need to setup these secrets to make them work in a production environment.
And also few lines about how you can test the integration locally. ( this would also help me to do a test as a user and validate the work you have done)